python/configurewindow/ConfigureWindow.py

```diff
@@ -75,7 +75,7 @@ def __init__(self,parent,id,title,shortcut, isPrefix = False):
         self.Bind(wx.EVT_BUTTON, self.DeletePrefix, id=1002)
 
 
-        self.list_game.SetSpacing(0)
+        #self.list_game.SetSpacing(0)
```

Comment:
This is technically a bug in wxPython that SetSpacing is missing.  (It wasn't removed intentionally.)  If you want I can fix the Fedora wxPython package so it's back.

Reply:
I agree there is inconsistency between code and documentation of wxPython. So yes you can fix it on the Fedora wxPython package.